build.gradle.kts

```diff
@@ -25,6 +27,15 @@ afterEvaluate {
   }
 }
 
+spotless {
+  kotlin {
+    target("**/*.kt")
+    ktlint("0.31.0").userData(mapOf("indent_size" to "2", "continuation_indent_size" to "2"))
```

Comment:
Hmm, shouldn't `continuation_indent_size` be 4?

Reply:
I agree, but FYI that'll nuke your 2space indented primary constructor args that we talked about on another PR

Reply:
Basically, yeah. [OkHttp also has that same size](https://github.com/square/okhttp/blob/master/build.gradle#L114) because of the primary constructor args.

Reply:
That sounds like a Spotless/KtLint bug

Reply:
Seems that IntelliJ does the same, so i'm not sure if it's a bug. 

<img width="657" alt="Screen Shot 2019-04-22 at 11 42 35 AM" src="https://user-images.githubusercontent.com/2227639/56518219-c9623d80-64f3-11e9-836f-2366b6c83e6c.png">

Notice the `multilineMethod` with 4 continuation indent.

Reply:
Weeeeird.. The official style guide says that class header params should use same indentation as the properties inside the class, and I believe that function headers follow the same rules: https://git.sqcorp.co/projects/CASH/repos/cash-android/pull-requests/8428/overview

Anyway, fine with leaving this as `2` as long as the code passes the checks. Might change later.

Reply:
Hmm that's interesting. I'll file a bug with ktlint soon and report it here.

Reply:
Fwiw I think it would be better to use 4 space for now since most everything else should be using that

Reply:
Would the check still pass?

Reply:
It will after the developer runs spotlessApply to autoformat :)

Reply:
Hmm looks like changing it to 4 doesn't do anything 🤷‍♂️ 

Some relevant threads:
[Seems like it was removed in 0.22](https://github.com/pinterest/ktlint/blob/8ddea8d60d75ced601cd7c6d44ecff0b971e07eb/CHANGELOG.md#0220---2018-04-22)

Reply:
Should we just remove the rule then? I think it'll confuse people going forward.

Reply:
5bb2a5d13847b355d93e147f321caa42790fa103

Tested by creating a new class like so:
```kotlin
class SampleClass(
    val something: String
)
```
which triggered a spotless violation. Upon running `./gradlew spotlessApply` it corrected to:
```kotlin
class SampleClass(
  val something: String
)
```

Reply:
Sweet, :shipit:!